packages/backend/src/services/UserService.ts

```diff
@@ -294,7 +294,9 @@ export class UserService extends BaseService {
         const [orgForUser] = await this.userModel.getOrganizationsForUser(
             userUuidToDelete,
         );
-        if (orgForUser.organizationUuid) {
+        // The user might not have an org yet
+        // This is expected on the "Cancel registration" flow on single org instances.
+        if (orgForUser?.organizationUuid) {
```

Comment:
Yes, we do the same on the frontend on `join organization` error

Reply:
One question though, maybe the boolean needs to be switched?

Reply:
mm, I don't think so ? 

this IF only makes sense if the user has an org. If not, we simply delete the user